src/Calendar.tsx

```diff
@@ -0,0 +1,13 @@
+import './Calendar.css';
```

Comment:
Every component and all related files should be in its separate folder. It helps to maintain the code in the future. When you will have more components and some of them will be bigger and also have more related files, like types, constants, utils etc. it will be impossible to find what you need.

Reply:
Changed the structure, plox check it :)